sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -841,6 +841,23 @@ abstract class JdbcDialect extends Serializable with Logging {
       metadata: MetadataBuilder): Unit = {}
 }
 
+/**
+ * Make the `classifyException` method throw out the original exception
+ */
+trait JdbcDialectHelper extends JdbcDialect {
```

Comment:
can we give it a clearer name? like `NoLegacyJDBCError`.

Reply:
Okay, let me update it.